app/dbOperations/problems.go

```diff
@@ -0,0 +1,154 @@
+package dbOperations
+
+import (
+	"errors"
+	"fmt"
+
+	gremcos "github.com/supplyon/gremcos"
+	"github.com/supplyon/gremcos/api"
+	"github.com/supplyon/gremcos/interfaces"
+
+	modelStorage "github.com/imeplusplus/dont-panic-api/app/modelStorage"
+)
+
+func CreateProblem(cosmos gremcos.Cosmos, problem modelStorage.Problem) (modelStorage.Problem, error) {
+	_, err := GetProblemByName(cosmos, problem.Name)
+
+	if err == nil {
+		return modelStorage.Problem{}, fmt.Errorf("there is already a problem with name %v", problem.Name)
+	}
+
+	g := api.NewGraph("g")
+
+	query := g.AddV("problem").Property("partitionKey", "problem")
+	query = addProblemVertexProperties(query, problem)
+
+	res, err := cosmos.ExecuteQuery(query)
+	if err != nil {
+		fmt.Println("Failed to execute a gremlin command", query.String())
+		return problem, err
```

Comment:
another tip here that might improve debugability:
consider using xerrors. This way you can have a builtin "stacktrace" of errors.
when you build an error like:
```
// imagine that err is fmt.Errorf("some stuff happened")
return xerrors.Errorf("loading some stuff: %w", err)
```
the final error returns "loading some stuff: some stuff happened"
and the xerrors.Is(err, target error) call returns true if either the error is either of both, so it's also useful for testing